application/src/main/java/org/opentripplanner/service/vehiclerental/model/RentalVehicleFuel.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.service.vehiclerental.model;
+
+import javax.annotation.Nullable;
+import org.opentripplanner.transit.model.basic.Distance;
+import org.opentripplanner.transit.model.basic.Ratio;
+
+/**
+ * Contains information about the current battery or fuel status.
+ * See the <a href="https://github.com/MobilityData/gbfs/blob/v3.0/gbfs.md#vehicle_statusjson">GBFS
+ * vehicle_status specification</a> for more details.
+ */
+public class RentalVehicleFuel {
+
+  /**
+   * Current fuel percentage, expressed from 0 to 1.
+   */
+  @Nullable
+  public final Ratio percent;
+
+  /**
+   * Distance that the vehicle can travel with the current fuel.
+   */
+  @Nullable
+  public final Distance range;
+
+  public RentalVehicleFuel(@Nullable Ratio fuelPercent, @Nullable Distance range) {
+    this.percent = fuelPercent;
+    this.range = range;
+  }
+
+  @Nullable
+  public Ratio getPercent() {
+    return this.percent;
+  }
+
+  @Nullable
+  public Distance getRange() {
```

Comment:
https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/doc/dev/decisionrecords/NamingConventions.md

We are far from consistent, but we don't add `get` to the getter, we just call them `range()` and `percent()`.